mne/gui/help/kit2fiff.json

```diff
@@ -1,7 +1,10 @@
 {
+  "hsp_file": "*.hsp or *.txt file containing digitized head shape points",
+  "fid_file": "*.elp or *.txt file containing 8 digitized fiducial points (\"electrode laser points\")",
```

Comment:
That's just how it's called in the traits gui already... :) technically aren't all 8 points fiducial points?